infrastructure/docker/Dockerfile-notebook

```diff
@@ -1,6 +1,9 @@
 ARG IMAGE_PY_VERSION=3.9
 FROM jupyter/base-notebook:python-$IMAGE_PY_VERSION
 
+USER 0
+RUN apt-get -y update && apt-get -y install gcc build-essential libopenblas-dev
```

Comment:
@akihikokuroda could you take a look if the `repository` application will require this kind of approach too? Apart from that as @psschwei said it built correctly on my side (x86) 😄

Reply:
@Tansito Thanks for review.  The repository doesn’t need to build wheel for qiskit-aer so it seems Ok.